crates/ruff_linter/src/rules/pycodestyle/snapshots/ruff_linter__rules__pycodestyle__tests__preview__W391_W391_2.py.snap

```diff
@@ -1,15 +1,15 @@
 ---
 source: crates/ruff_linter/src/rules/pycodestyle/mod.rs
-snapshot_kind: text
 ---
 W391_2.py:14:1: W391 [*] Too many newlines at end of file
    |
 12 |       foo()
 13 |       bar()
-14 | / 
-15 | | 
-16 | | 
-17 | | 
+14 | /
+15 | |
+16 | |
+17 | |
+   | |__^ W391
```

Comment:
This looks somewhat funny because we don't render new lines. I don't think it's something we have to address as part of this PR but might be good to be aware of